app/views/finders/_facet_collection.html.erb

```diff
@@ -7,7 +7,7 @@
     value: @results.user_supplied_keywords,
     id: "finder-keyword-search",
     controls: "js-search-results-info"
-  } if finder.show_keyword_search? %>
+  } if finder.show_keyword_search? and finder.name != 'All content' %>
   <div class="govuk-form-group" data-module="track-click" data-track-category="filterClicked"
```

Comment:
Nitpick: can we get a newline in here? I keep working on this file and briefly thinking the `if` on line 10 as relating to the code below.